src/components/sw360/AddAdditionalRoles/AddAdditionalRoles.tsx

```diff
@@ -87,9 +109,9 @@ function AddAdditionalRoles({ documentType, setDataInputList, inputList, setInpu
                                 </div>
                                 <div className='col-lg-5'>
                                     <input
-                                        name='email'
+                                        name='value'
```

Comment:
This may effect other components that are using this component

Reply:
@hoangnt2 , as we are using the data type to be InputKeyValue[] enum, so it won't effect what I believe.